src/planscape/planscape/settings.py

```diff
@@ -391,3 +392,5 @@
 CRONJOBS = [
     ("0 0 * * *", "planning.cron.delete_old_shared_links"),  # Runs at midnight daily
 ]
+
+REPORT_RECIPIENT_EMAIL = "no-reply@planscape.org"
```

Comment:
the nittiest of nits, but I think we we have this email already on a couple of places. Maybe we can have a NO_REPLY constant and use that instead?